addons/portal/wizard/portal_wizard.py

```diff
@@ -236,6 +236,10 @@ def _assert_user_email_uniqueness(self):
             ('id', '!=', self.user_id.id),
             ('login', '=ilike', email),
         ])
+        user = self._filter_user_email_uniqueness(user)
 
         if user:
             raise UserError(_('The contact "%s" has the same email has an existing user (%s).', self.partner_id.name, user.name))
+
+    def _filter_user_email_uniqueness(self, user):
+        return user
```

Comment:
- Missing Doc
- Isn't it possible to just patch the search domain used to find a similar user ?
=> If yes, it would be better to directly have something like `_get_similar_user_domain` here and overridden in website. 
=> If no, it would be better to have `_get_similar_user`

What do you think ?

Reply:
@Guillaume-gdi , thanks for your review! :pray: 
I adapted it according to your suggestion:
- I added a doc for the `_get_similar_user_domain()` method.
- I adapted this last function to patch the search domain rather than implementing a filter method; it is indeed more efficient as we only do one search compare to one search and a filter with the previous implementation :+1: